fragments/zjs.conf.dev

```diff
@@ -0,0 +1,17 @@
+# To include a module remove the # in front of it
+
+ZJS_EVENTS=y
+#ZJS_OCF=y
+ZJS_GPIO=y
+#ZJS_PERFORMANCE=y
+ZJS_PWM=y
+ZJS_UART=y
+ZJS_BLE=y
+ZJS_AIO=y
+ZJS_I2C=y
+#ZJS_GROVE_LCD=y
+ZJS_ARDUINO101_PINS=y
+ZJS_TIMERS=y
+ZJS_BUFFER=y
+#ZJS_SENSOR=y
+ZJS_CONSOLE=y
```

Comment:
Alphabetize these to make it easy to search for one and detect whether one is already in the list.